app.R

```diff
@@ -4,18 +4,31 @@ library(shinydashboard)
 library(shinyBS)
 library(shinyWidgets)
 library(boastUtils)
-library(shinyjs)
-library(data.table)
+# library(shinyjs) 
+## Fix: I don't believe that shinyjs is actively being used anywhere in the app
+# library(data.table)
+## Fix: The question banks you're loading are not large enough to require the use of
+## the data.table package and fread. Switch to read.csv
 
```

Comment:
@neilhatfield Hi Dr. Hatfield, So I tried changing this from fread to read.csv and whenever I do this, the first picture in the set of cards does not render and instead just says something like "7-spades.png". Should I keep it as fread? Or do you know why maybe this is occuring?

Reply:
@shravanisamala  I'm unable to replicate what you're describing. I used `read.csv(file = "www/pokerquestionbank.csv")`, `read.csv(file = "www/pokerquestionbank.csv", header = TRUE)`, and `read.csv(file = "pokerquestionbank")` (after moving the file to the main level). All of them resulted in full sets of cards rendering (including those with the question mark cards); that is, no broken images or just text listed.

Reply:
@neilhatfield Hi Dr. Hatfield, It is still happening to me so maybe we can take a look at it this Friday during the meeting. Thanks!

Reply:
Sounds like a plan